src/overlays/actors/ovl_En_Stream/z_en_stream.c

```diff
@@ -14,10 +14,12 @@ void EnStream_Init(Actor* thisx, GlobalContext* globalCtx);
 void EnStream_Destroy(Actor* thisx, GlobalContext* globalCtx);
 void EnStream_Update(Actor* thisx, GlobalContext* globalCtx);
 void EnStream_Draw(Actor* thisx, GlobalContext* globalCtx);
+void EnStream_WaitForPlayer(EnStream* this, GlobalContext* globalCtx);
+void EnStream_SuckPlayer(EnStream* this, GlobalContext* globalCtx);
+s32 func_80B0B81C(PosRot* vortexPosRot, PosRot* playerPosRot, Vec3f* v, f32 vortexYScale);
```

Comment:
We generally don't forward declare functions unless necessary, so you can omit these ones since they aren't used before their declaration.
```suggestion
```
```suggestion
```

Reply:
I can definitely remove the EnStream_SuckPlayer forward declaration, but removing the func_80B0B81C declaration causes functions that call it to no longer match.

Reply:
Oh it's a non matching, then I would leave the prototype in the #else next to the GLOBAL_ASM pragma since it's only relevant while it's a non matching